### DISCUSSION TITLE
Sonoff with modbus tcp/ip

### DISCUSSION THREAD

**Discussion by Pante90** [indomus/forum#1521] [Hardware & Firmware]
Buongiorno, 
vorrei chiedere se qualcuno sa dirmi se è possibile far comunicare i moduli sonoff con il protocollo Modbus Tcp/ip. Posso dirvi che il protocollo modbus tcp dialoga sulla porta 502. Se fosse possibile riflashare o se con qualche modalità particolare per far lavorare questi dispositivi con questo protocollo. 
ringrazio in anticipo l'auito...

**Comment by fizzi** [comment]
Ciao,

puoi essere più dettagliato ? ... A che moduli sonoff ti riferisci ?

**Comment by Pante90** [comment]
Moduli, ad esempio MINI, TH16....

**Reply by vinny74man** [reply]
Penso che devi leggere nelle caratteristiche del dispositivo che hai.
Se è predisposto per il modbus dovresti comunque avere porta rs485 quindi ben identificata.
